android/src/main/java/kjd/reactnative/bluetooth/RNBluetoothClassicModule.java

```diff
@@ -562,7 +562,7 @@ public void accept(Promise promise) {
    */
   @ReactMethod
   public void cancelAccept(Promise promise) {
-    if (mBluetoothAdapter == null || mBluetoothAdapter.isEnabled()) {
+    if (mBluetoothAdapter == null || !mBluetoothAdapter.isEnabled()) { //changed here
```

Comment:
@sanketch Look at this bug lol took like a couple hours of digging and figuring what was going on to isolate this issue.